signalstickers_client/classes/downloader.py

```diff
@@ -40,10 +36,14 @@ async def get_pack_metadata(http, pack_id, pack_key):
     Parse the pack manifest, and return
     a `StickerPack` object
     """
-    manifest_req = await http.get(CDN_MANIFEST_URL.format(
+    manifest_resp = await http.get(CDN_MANIFEST_URL.format(
         pack_id=pack_id), timeout=None)
-    manifest_req.raise_for_status()
-    manifest_encrypted = bytes(manifest_req.content)
+    if manifest_resp.status_code == 404:
```

Comment:
It looks like Signal returns a `403` when a pack is not found ([valid pack](https://cdn.signal.org/stickers/d9ae4ca540a80a00843660854e6130d2/manifest.proto) - [invalid pack](https://cdn.signal.org/stickers/d9ae4ca540a80a00843660854e6130d1/manifest.proto))